Documentation/HowTos/ExtendingPages.rst

```diff
@@ -32,7 +32,7 @@ in the below example:
 Fusion (Sites/Vendor.Site/Resources/Private/Fusion/Root.fusion) ::
 
 	prototype(Neos.Neos:Page) {
-		backgroundImage = ${q(node).property('backgroundImage')}
+		body.backgroundImage = ${q(node).property('backgroundImage')}
```

Comment:
would be better to have a proper example that converts the image into a `url('...')` string

Reply:
my usecase was a little bit different then the one in docs. I've used image related to page to enrich the megamenu. So I've used the media image viewhelper from within the menu template. 
In general I think putting image in background is less common then just rendering the image. 

Anyway, the most important part is to fix the fatal error. Most users can figure it out from there.

Reply:
sure however the example is for setting a background image, regardless of that being the less common case of adding a picture to a page template..

Reply:
@aertmann If you're asking me to modify the pull request I'm sorry to say I don't have time to do it in the next weeks. So please accept the pull request as is or modify it yourself. Thanks.